odoo/addons/test_lint/tests/test_i18n.py

```diff
@@ -0,0 +1,129 @@
+import logging
+import re
+
+from . import lint_case
+
+from odoo import tools
+
+_logger = logging.getLogger(__name__)
+
+
+class TestI18n(lint_case.LintCase):
+    # Assumption: Text starting with an uppercase letter is likely supposed to be translated.
+    PROPS_RE = re.compile(
+    r"""
+        (
+            <[A-Z][^>]+\s+                  # Match a component opening tag with a capital letter, followed by attributes
```

Comment:
"followed by attributes" → this line is not looking for attributes

rewording suggestion:
Match the opening tag of a component node. We assume that tags starting with an uppercase letter refer to a component